plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -165,6 +162,7 @@
     public static final String OPENSHIFT_DEPLOYMENT_LABEL               = "deployment";
     public static final String CHE_MOUNTED_WORKSPACE_FOLDER             = "/workspace-logs";
     public static final String WORKSPACE_LOGS_FOLDER_SUFFIX             = "-logs";
+    public static final int OPENSHIFT_WAIT_POD_TIMEOUT                  = 240;
```

Comment:
It may be worth it to leave this private and just have a similar field in `OpenShiftDeploymentCleaner`. There's no reason the timeout on scaling down deployments *needs* to be the same as the one for waiting for containers to start (in fact, we may want the timeout for scaling down to be less than 4 minutes, that seems like a long time). Ideally, the scale down happens in under 5 seconds.